cypress/component/AllServicesDropdown/AllServicesDropdown.cy.tsx

```diff
@@ -23,7 +23,7 @@ describe('<AllServicesDropdown />', () => {
   it('should close all services dropdown in link matches current pathname', () => {
     function checkMenuClosed() {
       cy.get('.pf-v5-c-menu-toggle__text').click();
-      cy.contains('All services').should('exist');
```

Comment:
There is also a `cy.contains('All services').should('not.exist');` bellow. This check has to be changed because now its always true. The test does not verify if the dropdown was closed or not.